source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,118 @@
+---
+title: RSpec 3.6 has been released!
+author: Sam Phippen, TODO: other editors
+---
+
+RSpec 3.6 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be an easy
+upgrade for anyone already using RSpec 3, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes over XXX commits and YYY
+merged pull requests from over 50 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+### Core: Errors outside examples now handled and formatted well
+
+In previous versions of RSpec, we allowed errors encountered while loading spec
+files or running `:suite` hooks to crash the ruby interpreter, giving you its
+default full-stacktrace output.  In RSpec 3.6, we now handle all errors that
+occur outside examples, and format them nicely including a filtered backtrace
+and a code snippet for the site of the error.  For example, an error in a
+`before(:suite)` hook is now formatted like this:
+
+<img width="479" alt="screen shot 2017-04-27 at 9 59 12 pm"
+src="https://cloud.githubusercontent.com/assets/49391/25514870/cb9db6c8-2b94-11e7-952f-f26fd783512b.png">
+
+Thanks to Jon Rowe for assisting with this feature.
+
+### Core: `config.fail_if_no_examples`
+
+As it currently stands RSpec will exit with code `0` indicating success if no
+examples are defined. This configuration option makes it possible to cause RSpec
+to exit with code `1` indicating failure. This is useful in CI environments, it
+helps detect when you've misconfigured RSpec to look for specs in the wrong
+place or with the wrong pattern. When this occurs we consider finding no specs
```

Comment:
> When this occurs

What is the "this" here?  Do you mean when RSpec is misconfigured to look for example in the wrong place?  Or when the user has configured it with `config.fail_if_no_examples = true`?  Both are needed for the rest of the statement to be true but this sentence doesn't really explain that.